xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -1315,6 +1374,172 @@ private void cleanUpResourceTimers() {
     }
   }
 
+  private void handleResourcesAcked(
+      ResourceType type, Map<String, ParsedResource> parsedResources, String version,
+      String nonce, boolean callOnAbsent) {
```

Comment:
Based on this PR, `callOnAbsent` is true for LDS/CSD, and is false for RDS/EDS. Don't we need to set it to true for LDS/RDS and false for RDS/EDS? If a resource is not presented in LDS/RDS, we considered it implicitly removed.

Reply:
Sorry, I made a mistake here. LDS/CDS are SotW, and RDS/EDS are incremental. https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol#four-variants.